### PR TITLE
V6 Components Inputs

### PR DESCRIPTION
**Pull request by ryandialpad**
Inputs Feature Merge

### CODE REVIEW

**Comment by ryandialpad** [comment]
@dialpad-drew There were a large number of merge conflicts so I am adding you as a reviewer to make sure nothing was missed.

File: docs/components/input.html [hunk 1]
```diff
@@ -3,149 +3,413 @@
 title: Input
 description: An input field is an input control that allows users to enter alphanumberic information. It can have a range of options and supports single line and multi-line lengths, as well as varying formats, including numbers, masked passwords, etc.
 ---
```
Comment (ryandialpad) [inline docs/components/input.html:5]: @dialpad-drew Is this line correct?

Comment (dialpad-drew) [reply docs/components/input.html:5]: Yea. I agree it's slightly redundant, but I'm trying to follow this pattern for form elements:

> A checkbox is an input control that allows users to select zero, one, or more options from a number of choices.

> A select menu is an input control that allows users to choose one option from a list.

> A radio is an input control that allows users to select only one option from a number of choices.

**Comment by dialpad-drew** [comment]
> @dialpad-drew There were a large number of merge conflicts so I am adding you as a reviewer to make sure nothing was missed.

Just went through this. Ironically, everything looks as expected _except_ Input documentation. Looking into it.

**Comment by dialpad-drew** [comment]
> > @dialpad-drew There were a large number of merge conflicts so I am adding you as a reviewer to make sure nothing was missed.
> 
> Just went through this. Ironically, everything looks as expected _except_ Input documentation. Looking into it.

@ryandialpad Got Input documentation back. Give that a once-over once the preview build is ready if you would please.

**Pull request merged**